From 8cfa18fb5fe7b181f6c5b28083f4ba6fd1af5217 Mon Sep 17 00:00:00 2001
From: Dirk Tilger <dirk@systemication.com>
Date: Mon, 2 Feb 2026 10:10:10 +0100
Subject: [PATCH] Remove embedded dependencies for system library build

---
 Makefile.am  | 15 ---------------
 configure.ac |  1 -
 2 files changed, 16 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index cc03c67..fa00cdb 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,15 +1,12 @@
 ACLOCAL_AMFLAGS = -I m4
-
 SUBDIRS = \
 	include \
 	common \
-	libcerror \
 	libfdatetime \
 	po \
 	tests \
 	manuals \
 	msvscpp
-
 DPKG_FILES = \
 	dpkg/changelog \
 	dpkg/changelog.in \
@@ -20,24 +17,19 @@ DPKG_FILES = \
 	dpkg/libfdatetime-dev.install \
 	dpkg/libfdatetime.install \
 	dpkg/source/format
-
 GETTEXT_FILES = \
 	config.rpath \
 	po/Makevars.in
-
 PKGCONFIG_FILES = \
 	libfdatetime.pc.in
-
 SPEC_FILES = \
 	libfdatetime.spec \
 	libfdatetime.spec.in
-
 EXTRA_DIST = \
 	$(DPKG_FILES) \
 	$(GETTEXT_FILES) \
 	$(PKGCONFIG_FILES) \
 	$(SPEC_FILES)
-
 DISTCLEANFILES = \
 	config.status \
 	config.cache \
@@ -47,20 +39,13 @@ DISTCLEANFILES = \
 	Makefile \
 	Makefile.in \
 	po/Makevars
-
 pkgconfigdir = $(libdir)/pkgconfig
-
 pkgconfig_DATA = \
 	libfdatetime.pc
-
 libtool: @LIBTOOL_DEPS@
 	cd $(srcdir) && $(SHELL) ./config.status --recheck
-
 lib: library
-
 library:
 	(cd $(srcdir)/common && $(MAKE) $(AM_MAKEFLAGS))
-	(cd $(srcdir)/libcerror && $(MAKE) $(AM_MAKEFLAGS))
 	(cd $(srcdir)/libfdatetime && $(MAKE) $(AM_MAKEFLAGS))
 	(cd $(srcdir)/po && $(MAKE) $(AM_MAKEFLAGS))
-
diff --git a/configure.ac b/configure.ac
index c08f9e8..8c3ebbc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -91,7 +91,6 @@ dnl Generate Makefiles
 AC_CONFIG_FILES([Makefile])
 AC_CONFIG_FILES([include/Makefile])
 AC_CONFIG_FILES([common/Makefile])
-AC_CONFIG_FILES([libcerror/Makefile])
 AC_CONFIG_FILES([libfdatetime/Makefile])
 AC_CONFIG_FILES([po/Makefile.in])
 AC_CONFIG_FILES([po/Makevars])
-- 
2.49.1

