From e93597906cea75ee65824fbd8f753e1d22aca43c Mon Sep 17 00:00:00 2001
From: Dirk Tilger <dirk@systemication.com>
Date: Mon, 2 Feb 2026 08:58:35 +0100
Subject: [PATCH] Remove embedded libcerror for system library build

---
 Makefile.am  | 2 --
 configure.ac | 1 -
 2 files changed, 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 8984080..a7a8c82 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -3,7 +3,6 @@ ACLOCAL_AMFLAGS = -I m4
 SUBDIRS = \
 	include \
 	common \
-	libcerror \
 	libcthreads \
 	po \
 	manuals \
@@ -60,7 +59,6 @@ lib: library
 
 library:
 	cd $(srcdir)/common && $(MAKE) $(AM_MAKEFLAGS)
-	cd $(srcdir)/libcerror && $(MAKE) $(AM_MAKEFLAGS)
 	cd $(srcdir)/libcthreads && $(MAKE) $(AM_MAKEFLAGS)
 	cd $(srcdir)/po && $(MAKE) $(AM_MAKEFLAGS)
 
diff --git a/configure.ac b/configure.ac
index e4ac342..122a17a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -91,7 +91,6 @@ dnl Generate Makefiles
 AC_CONFIG_FILES([Makefile])
 AC_CONFIG_FILES([include/Makefile])
 AC_CONFIG_FILES([common/Makefile])
-AC_CONFIG_FILES([libcerror/Makefile])
 AC_CONFIG_FILES([libcthreads/Makefile])
 AC_CONFIG_FILES([po/Makefile.in])
 AC_CONFIG_FILES([po/Makevars])
-- 
2.49.1

