From 79543a1d61bac3b9f00a231ddb9204ef2c694326 Mon Sep 17 00:00:00 2001
From: Dirk Tilger <dirk@systemication.com>
Date: Mon, 2 Feb 2026 09:18:11 +0100
Subject: [PATCH] Remove embedded dependencies for system library build

---
 Makefile.am  | 21 ---------------------
 configure.ac |  4 ----
 2 files changed, 25 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index b5db608..95caf7c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,18 +1,12 @@
 ACLOCAL_AMFLAGS = -I m4
-
 SUBDIRS = \
 	include \
 	common \
-	libcerror \
-	libclocale \
-	libcnotify \
-	libuna \
 	libcfile \
 	po \
 	manuals \
 	tests \
 	msvscpp
-
 DPKG_FILES = \
 	dpkg/changelog \
 	dpkg/changelog.in \
@@ -23,24 +17,19 @@ DPKG_FILES = \
 	dpkg/libcfile-dev.install \
 	dpkg/libcfile.install \
 	dpkg/source/format
-
 GETTEXT_FILES = \
 	config.rpath \
 	po/Makevars.in
-
 PKGCONFIG_FILES = \
 	libcfile.pc.in
-
 SPEC_FILES = \
 	libcfile.spec \
 	libcfile.spec.in
-
 EXTRA_DIST = \
 	$(DPKG_FILES) \
 	$(GETTEXT_FILES) \
 	$(PKGCONFIG_FILES) \
 	$(SPEC_FILES)
-
 DISTCLEANFILES = \
 	config.status \
 	config.cache \
@@ -50,23 +39,13 @@ DISTCLEANFILES = \
 	Makefile \
 	Makefile.in \
 	po/Makevars
-
 pkgconfigdir = $(libdir)/pkgconfig
-
 pkgconfig_DATA = \
 	libcfile.pc
-
 libtool: @LIBTOOL_DEPS@
 	cd $(srcdir) && $(SHELL) ./config.status --recheck
-
 lib: library
-
 library:
 	cd $(srcdir)/common && $(MAKE) $(AM_MAKEFLAGS)
-	cd $(srcdir)/libcerror && $(MAKE) $(AM_MAKEFLAGS)
-	cd $(srcdir)/libclocale && $(MAKE) $(AM_MAKEFLAGS)
-	cd $(srcdir)/libcnotify && $(MAKE) $(AM_MAKEFLAGS)
-	cd $(srcdir)/libuna && $(MAKE) $(AM_MAKEFLAGS)
 	cd $(srcdir)/libcfile && $(MAKE) $(AM_MAKEFLAGS)
 	cd $(srcdir)/po && $(MAKE) $(AM_MAKEFLAGS)
-
diff --git a/configure.ac b/configure.ac
index 655d2e9..7410c87 100644
--- a/configure.ac
+++ b/configure.ac
@@ -109,10 +109,6 @@ dnl Generate Makefiles
 AC_CONFIG_FILES([Makefile])
 AC_CONFIG_FILES([include/Makefile])
 AC_CONFIG_FILES([common/Makefile])
-AC_CONFIG_FILES([libcerror/Makefile])
-AC_CONFIG_FILES([libclocale/Makefile])
-AC_CONFIG_FILES([libcnotify/Makefile])
-AC_CONFIG_FILES([libuna/Makefile])
 AC_CONFIG_FILES([libcfile/Makefile])
 AC_CONFIG_FILES([po/Makefile.in])
 AC_CONFIG_FILES([po/Makevars])
-- 
2.49.1

